platforms/gke/base/core/custom_compute_class/templates/manifests/gpu/g4-180gb/custom-compute-gpu-g4-180gb-s48-x1.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: cloud.google.com/v1
+kind: ComputeClass
+metadata:
+  name: gpu-g4-180gb-s48-x1
```

Comment:
If following the other naming conventions, this should be `rtx-pro-6000-96gb-x1`. There aren't any machine type variants for the g4s like there are for the g2s.

These should all go in a folder `rtx-pro-6000-96gb`